src/devicecontroller/DefaultDeviceController.ts

```diff
@@ -165,6 +165,25 @@ export default class DefaultDeviceController implements DeviceControllerBasedMed
     }
 
     const context = DefaultDeviceController.getAudioContext();
+
+    if (context instanceof OfflineAudioContext) {
+      // Nothing to do.
```

Comment:
nit: Do we need this if statement at all?

Reply:
We need the `else` that comes after it 😂 

I find it's clearer to write `instanceof` in the positive form and explain that the block is intentionally empty than to write `if (!(context instanceof OfflineAudioContext)) {` with a comment above.